Cargo.toml

```diff
@@ -12,20 +12,18 @@ readme = "README.md"
 
 
 [features]
-default = [ "secp-recovery" ]
```

Comment:
It seems like this was redundant all along? It seems like it was introduced by @stevenroose not long ago in f7422fb8d12714884533981eeb0b127240c9230c. I'd like to know if there's some arcane magic going on that requires this approach which we are breaking now.

Reply:
I thought about it for a while but to my knowledge, it is redundant.

We would have to remove the `recovery` feature from the dependency declaration for this feature to do anything meaningful.

Thanks for linking to the commit! I forgot to check for `cfg` attributes that are using this feature, the will have to be removed too.

I am gonna push a `fixup!` commit for now so that it is consistent until there is a decision on what to do about it.